src/Container/WhoopsErrorResponseGeneratorFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Container;
+
+use Interop\Container\ContainerInterface;
+use Zend\Expressive\Middleware\WhoopsErrorResponseGenerator;
+
+class WhoopsErrorResponseGeneratorFactory
+{
+    /**
+     * @param ContainerInterface $container
+     * @return WhoopsErrorResponseGenerator
+     */
+    public function __invoke(ContainerInterface $container)
+    {
+        return new WhoopsErrorResponseGenerator(
+            $container->get('Zend\Expressive\Whoops')
```

Comment:
No, because it's a virtual (vs defined) class name, and resolves to a `Whoops\Run` instance. Having it as a string helps signal that fact.